staging/src/k8s.io/cluster-bootstrap/token/util/helpers.go

```diff
@@ -1,136 +0,0 @@
-/*
```

Comment:
the changes in this file - moving publicly exposed code to a different package would be breaking to users who import "k8s.io/cluster-bootstrap/token/util":
https://grep.app/search?current=2&q=%22k8s.io/cluster-bootstrap/token/util%22
as you can see there are a number of projects that do that.

the package structure of this repository is unfortunate, seems like the result of a best effort, where contributors have amended on action items over time, resulting in multiple "utils" folders.

despite that, i think at this point we should preserve the same structure to not break users.
a migration plan is possible where we duplicate publicly exposed code in this package and the target package, deprecate the old package code, but ideally remove it in a MAJOR release of this library (e.g. v2). it currently uses the staging process, which makes this difficult, IMO.